client/source/js/modules/ui/modal/modal-service.js

```diff
@@ -21,15 +21,24 @@ define([
          * executes the right callback depending on the user's choice.
          */
         confirm: function (onAccepted, onRejected, acceptButton, rejectButton, message, title) {
-          $modal.open({
+
+          var onModalKeyDown = function (anEvent) {        
```

Comment:
The convention of using `a` or `an` or `someBlah` is explicitely referring to an instance(s). When used consistently, it makes argument naming less ambiguous.